src/app/components/account-details/account-details.component.ts

```diff
@@ -72,6 +76,32 @@ export class AccountDetailsComponent implements OnInit, OnDestroy {
     });
 
     await this.loadAccountDetails();
+    this.shouldLoop = true;
+    this.updateList();
+  }
+
+  // check for new pending in any account every 2sec and update list if needed
+  async updateList() {
```

Comment:
Yeah so there are probably 100 ways to do this but I wanted to start with something that works and maybe enhance later. Using this asynchronous method of polling regularly has two advantages as I see it.

1. If you receive 1000 transactions during those 5sec, you only have to call the RPC one time instead of 1000 if it would be a direct connection to the websocket. And the reason for checking pending via RPC is because of 2.
2. By calling RPC and check for pending you get less reliable on the webssocket. It's very important this list is correct and relying on only websocket would make it very easy to miss one transaction if the websocket disconnect just for a few seconds (on either Nault or the backend). The websocket does NOT recover missing transactions. If that would happen now, the list would be completely unaffected and you will always get ALL the missed transactions as soon as the wallet is updated (with an average of 2.5sec). A websocket generally is a really bad solution and doesn't scale very well. There is a reason TIX wallet does not use websocket at all because it's unreliable. So that's the underlying reason for this polling method. You think OK to have for now until we or someone else come up with something better?

Reply:
My issue is that I want to reduce the calls to the API as much as possible. It's correct that you can miss transactions from the websocket, but then this function should only work as a fallback and should not be called that much to reduce stress on the API. But 5 seconds seems reasonable.

Reply:
Yeah, it's been reworked and function much better now. No extra API calls.